src/parsersettings.h

```diff
@@ -13,6 +13,6 @@ extern int parser_error_pos;
 void parser_init();
 
 fs::path search_libs(const fs::path &localpath);
-fs::path find_valid_path(const fs::path &sourcepath, 
-                         const fs::path &localpath, 
-                         const std::vector<std::string> *openfilenames = nullptr);
+fs::path find_valid_path(const fs::path &sourcepath,
+												 const fs::path &localpath,
+												 const std::vector<std::string> *openfilenames = nullptr);
```

Comment:
This indent seems a little large.

Reply:
It's because the github web viewer default to a tab size of 8, while i use 2.

Reply:
..which is one of the reasons why we shouldn't have used tab for indenting in the first place : /

Reply:
https://github.com/openscad/openscad/blob/e1c1ba37fa717f882cc44a7eac70be9a83a1dd7e/src/parsersettings.h?ts=2